app/locale/en.coco.coffee

```diff
@@ -3574,3 +3575,5 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     share_info: 'To give other teachers access to the class, add their emails below.'
     class_owner: 'Class Owner'
     share: 'Share'
+    support_coco: 'support@codecombat.com',
```

Comment:
We shouldn't translate this; the email address will be the same across languages. If we want to make this into a dynamic variable, putting it in to some other shared variables file (like app/core/utils.coffee) would be better.

Reply:
Done. (Added to utils)